pycodestyle.py

```diff
@@ -447,11 +447,11 @@ def whitespace_around_keywords(logical_line):
     E274: True\tand False
     """
     for match in KEYWORD_REGEX.finditer(logical_line):
-        before, after = match.groups()
+        before, kw, after = match.groups()
```

Comment:
I suggest using the more explicit `keyword` instead of `kw` for this variable name.

Reply:
That would shadow the global "keyword" module name.

Reply:
Ah, that's unfortunate. I missed this due to only reading the diff.